cpp/arcticdb/processing/clause.cpp

```diff
@@ -671,7 +673,24 @@ Composite<EntityIds> MergeClause::process(Composite<EntityIds>&& entity_ids) con
     size_t max_end_row = 0;
     size_t min_start_col = std::numeric_limits<size_t>::max();
     size_t max_end_col = 0;
-    procs.broadcast([&input_streams, &min_start_row, &max_end_row, &min_start_col, &max_end_col](auto&& proc) {
+
+    bool stream_attributes_set = false;
+    stream::Index index = stream::TimeseriesIndex::default_index();
+    StreamId stream_id;
+    StreamDescriptor stream_descriptor;
+
+    procs.broadcast([&stream_attributes_set, &index, &stream_id, &stream_descriptor, &input_streams, &min_start_row, &max_end_row, &min_start_col, &max_end_col](auto&& proc) {
+        if (!stream_attributes_set) {
+            // TODO: is there a nicer way to get those pieces of information at the moment?
```

Comment:
There's no guarantee that would be the case. We might be merging entirely different symbols